File: solutions/beecrowd/1805/1805.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);

    while (sc.hasNextLong()) {
        val first = sc.nextLong()
        val last = sc.nextLong()
        println((last - first + 1) * (first + last) / 2)
    }
}

File: solutions/beecrowd/1805/1805.scala
import java.util.Scanner

object Main {
  def main(args: Array[String]): Unit = {
    val scanner = new Scanner(System.in)

    while (scanner.hasNextLong) {
      val first = scanner.nextLong()
      val last = scanner.nextLong()
      val result = (last - first + 1) * (first + last) / 2

      println(result)
    }
  }
}
